packages/manager/modules/pci/src/projects/project/storages/object-storage/object-storage.controller.js

```diff
@@ -7,6 +7,7 @@ export default class PciStoragesObjectStorageController {
 
   $onInit() {
     this.loadMessages();
+    this.isUserTabActive = this.currentActiveLink().includes('users');
```

Comment:
Are you sure that we need not evaluate this again? The active tab might change after this code executes right? :)

Reply:
Have moved this logic to resolve,